ui/app/views/deployments/new.html

```diff
@@ -19,7 +19,8 @@ <h1 class="page-header">New Deployment</h1>
                         <table class="table table-hover" datatable="ng" dt-options="dtOptions">
                             <thead>
                             <tr>
-                                <th ng-if="!isGroupDeployment"></th>
```

Comment:
What happened to the "isGroupDeployment"?

Reply:
It isn't right to remove a header (ng-if) because it may break the table display, the header should always exist and the child should be hidden \ empty in case of group deployment